data.rb

```diff
@@ -0,0 +1,24 @@
+require 'json'
+
+def save_album(album)
+  JSON.dump(album, File.open('./storage/albums.json', 'w+'))
+end
+
+def save_genre(genre)
+  JSON.dump(genre, File.open('./storage/genre.json', 'w+'))
+end
+
+def save_labels(label)
+  JSON.dump(label, File.open('./storage/label.json', 'w+'))
+end
+
+def save_source(source)
+  JSON.dump(source, File.open('./storage/source.json', 'w+'))
+end
+
+def save_data(item)
+  save_album(item.music_albums)
+  save_genre(item.genres)
+  save_source(item.sources)
+  save_labels(item.labels)
+end
```

Comment:
- [ ] When I try to create a new game, movie, or music album... after I removed the `storage` folder the app crashes, kindly consider fixing this so the app saves the data even when the folder is messing

![Screenshot_20221126_133347](https://user-images.githubusercontent.com/25227022/204089472-fda4e579-28ac-47da-b5db-b266c89542d7.png)

![Screenshot_20221126_133504](https://user-images.githubusercontent.com/25227022/204089475-ce32d1e1-da46-4eb7-acff-748a64b151c5.png)